pallets/msa/src/tests.rs

```diff
@@ -15,12 +15,12 @@ use common_primitives::{
 };
 use common_runtime::extensions::check_nonce::CheckNonce;
 use frame_support::{
-	assert_err, assert_noop, assert_ok,
+	assert_err, assert_err_with_weight, assert_noop, assert_ok,
```

Comment:
I did not know about this one `assert_err_with_weight` but it looks like we are not using it.

Reply:
Removed